stable/datadog/values.yaml

```diff
@@ -261,6 +261,35 @@ datadog:
   #   exec:
   #     command: ["/bin/true"]
 
+  ## @param system_probe - object - required
+  ## Enable system_probe agent and provide custom configs
+  #
+  system_probe:
+    ## @param enabled - boolean - required
+    ## Set this to true to enable system-probe agent
+    #
+    enabled: false
+    ## @param debug_port - integer - optional
+    ## Specify the port to expose pprof and expvar for system-probe agent
+    #
+    # debug_port: 7654
+
+    ## @param sysprobe_socket - string - required
+    ## Set the unix socket directory for system-probe data to be read by process-agent
+    #
+    sysprobe_socket: /opt/datadog-agent/run
```

Comment:
nit: I would call this `sysprobe_socket` to indicate it's a directory and not the full path to the socket.

Also, does this need to be configurable at all?  I don't think there's any reason to change it since only the core agent and  sysprobe need to agree on the location.

Reply:
I'm just about to hard-code it in configmap instead, I also realize this location doesn't need to be configured.

Reply:
Actually it cannot be changed, because we cannot inject system-probe.yaml to datadog-agent container for now, so process-agent has to use the default location, which is ☝️

Reply:
Er, `sysprobe_socket_dir`, but it looks like you got rid of it.

Reply:
Yea because it's no longer configurable :)